core/src/main/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -135,105 +166,133 @@
   public static final DistributionViewDescriptor RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW =
       DistributionViewDescriptor.create(
-          "rpc client roundtrip_latency",
+          "grpc.io/client/roundtrip_latency/distribution_cumulative",
           "Latency in msecs",
           RPC_CLIENT_ROUNDTRIP_LATENCY,
           DistributionAggregationDescriptor.create(RPC_MILLIS_BUCKET_BOUNDARIES),
           Arrays.asList(RPC_CLIENT_METHOD));
   public static final DistributionViewDescriptor RPC_CLIENT_SERVER_ELAPSED_TIME_VIEW =
       DistributionViewDescriptor.create(
-          "rpc client server_elapsed_time",
+          "grpc.io/client/server_elapsed_time/distribution_cumulative",
           "Server elapsed time in msecs",
           RPC_CLIENT_SERVER_ELAPSED_TIME,
           DistributionAggregationDescriptor.create(RPC_MILLIS_BUCKET_BOUNDARIES),
           Arrays.asList(RPC_CLIENT_METHOD));
   public static final DistributionViewDescriptor RPC_CLIENT_REQUEST_BYTES_VIEW =
       DistributionViewDescriptor.create(
-          "rpc client request_bytes",
+          "grpc.io/client/request_bytes/distribution_cumulative",
           "Request bytes",
           RPC_CLIENT_REQUEST_BYTES,
           DistributionAggregationDescriptor.create(RPC_BYTES_BUCKET_BOUNDARIES),
           Arrays.asList(RPC_CLIENT_METHOD));
   public static final DistributionViewDescriptor RPC_CLIENT_RESPONSE_BYTES_VIEW =
       DistributionViewDescriptor.create(
-          "rpc client response_bytes",
+          "grpc.io/client/response_bytes/distribution_cumulative",
           "Response bytes",
           RPC_CLIENT_RESPONSE_BYTES,
           DistributionAggregationDescriptor.create(RPC_BYTES_BUCKET_BOUNDARIES),
           Arrays.asList(RPC_CLIENT_METHOD));
   public static final DistributionViewDescriptor RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES_VIEW =
       DistributionViewDescriptor.create(
-          "rpc client uncompressed_request_bytes",
+          "grpc.io/client/uncompressed_request_bytes/distribution_cumulative",
           "Uncompressed Request bytes",
           RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES,
           DistributionAggregationDescriptor.create(RPC_BYTES_BUCKET_BOUNDARIES),
           Arrays.asList(RPC_CLIENT_METHOD));
   public static final DistributionViewDescriptor RPC_CLIENT_UNCOMPRESSED_RESPONSE_BYTES_VIEW =
       DistributionViewDescriptor.create(
-          "rpc client uncompressed_response_bytes",
+          "grpc.io/client/uncompressed_response_bytes/distribution_cumulative",
           "Uncompressed Response bytes",
           RPC_CLIENT_UNCOMPRESSED_RESPONSE_BYTES,
           DistributionAggregationDescriptor.create(RPC_BYTES_BUCKET_BOUNDARIES),
           Arrays.asList(RPC_CLIENT_METHOD));
+  public static final DistributionViewDescriptor RPC_CLIENT_REQUEST_COUNT_VIEW =
+      DistributionViewDescriptor.create(
+          "grpc.io/client/request_count/distribution_cumulative",
+          "Count of client RPC requests",
```

Comment:
This distribution view is useful because there can be multiple requests/responses per RPC. The description should make it clear. Maybe "Count of request messages per client RPC".
Ditto for the rest three distribution views.

We probably also need to add interval views for the request response counts.